common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/NativeAuthOAuth2Configuration.kt

```diff
@@ -51,8 +50,9 @@ class NativeAuthOAuth2Configuration(
 
     companion object {
         //Base url for the mock API to make Native Auth calls. See the swagger at
-        // https://native-auth-mock-api.azurewebsites.net/doc#/ for all possible urls
-        private const val MOCK_API_URL_WITH_NATIVE_AUTH_TENANT = "https://native-auth-mock-api.azurewebsites.net/lumonconvergedps.onmicrosoft.com"
+        // $(MOCK_API_URL)/doc#/ for all possible urls
+        private val BASE_PATH = BuildValues.getMockApiUrlForNativeAuthAuthority()
```

Comment:
The same here. Frist use the concrete string to pass the pipeline since it's not a test file.